src/back-end/config.ts

```diff
@@ -54,14 +54,16 @@ export const ENV = NODE_ENV;
 
 const logger = makeDomainLogger(consoleAdapter, "back-end:config");
 
-export const LOG_DEBUG =
-  parseBooleanEnvironmentVariable(
-    get("LOG_DEBUG", ENV === "development" ? "1" : "0")
-  ) ?? false;
+export const LOG_LEVEL =
```

Comment:
I haven't tried, but is it possible to use a type annotation here as a LogLevel so that we don't need to use an assertion later on?